app/webpacker/lib/utils/competition-table.js

```diff
@@ -1,11 +1,33 @@
 import React from 'react';
 import { DateTime, Interval } from 'luxon';
 import I18n from '../i18n';
+import { countries } from '../wca-data.js.erb';
 
 function parseDateString(yyyymmddDateString) {
   return DateTime.fromFormat(yyyymmddDateString, 'yyyy-MM-dd');
 }
 
+export const competitionStatusText = (competition, registrationStatus) => {
```

Comment:
I think there's a space missing between the 2 pieces. (Note the first piece might not exist.)